www/doc/1.7/report.html

```diff
@@ -20,143 +22,176 @@ <h1>Herbie reports</h1>
+    generates HTML reports full of information about the accuracy of
+    your input and its output expressions.</p>
 
   <h2 id="summary">Summary numbers</h2>
 
   <p>First, a brief summary of the results. For most uses, the
-    “Average Error” number, which describes how accurate the input and
-    output expressions are, is the most important statistic in this
-    section. The other numbers list time Herbie took to improve the
-    program and the <a href="input.html#precisions">precision</a>
-    assumed for floating-point operations.
+    "Percentage Accurate" and "Speedup" metric, which describes the change in accuracy and speed up from
+    the starting expression to what Herbie thinks might be the best alterative,
+    is the most important statistics in this section. The other numbers list
+    time Herbie took to improve the program and number of other alternatives
+    found.
   </p>
 
   <figure>
     <img width="100%" src="report-large.png" />
-    <figcaption>Summary numbers from a Herbie report. The "binary64"
-    precision refers to double-precision IEEE-754
-    arithmetic.</figcaption>
+    <figcaption>Summary numbers from a Herbie report.</figcaption>
   </figure>
 
-  <h2 id="programs">Input and output programs</h2>
+  <ul>
+    <li><b>Percentage Accurate</b>: The change in accuracy from
+      the starting expression to what Herbie thinks might be the best alterative.</li>
+    <li><b>Time</b>: The time it took herbie to improve the expression.</li>
+    <li><b>Alternatives</b>: Number of alterative solutions found.</li>
+    <li><b>Speedup</b>: Relative speed of fastest alternative that improves accuracy.</li>
+  </ul>
+
+  <h2 id="spec">Specification</h2>
 
-  <p>Second, the input and output programs themselves, in standard
-  mathematical syntax. In the top-right corner, the drop-down can be
-  used to switch to C syntax or some other format.</p>
+  <p>Second, a summary of the specification that you gave to herbie. In the top
+    right you can change the notation of the expression from Math to other
+    syntax like FPCore or Python.</p>
+  <p>
+    <a href="input.html#preconditions">Preconditions</a> and other
+    <a href="http://fpbench.org/spec/fpcore-2.0.html#syntax" target="_blank">FPCore syntax</a> may be added to give
+    Herbie more
+    information about the expression you provided. Please see the
+    <a href="input.html#sec1">input page</a> for more information.
+  </p>
 
   <figure>
-    <img width="100%" src="report-prog.png" />
-    <figcaption>Input and output program from a Herbie report.</figcaption>
+    <img width="100%" src="specification.png" />
+    <figcaption>The bar shows the percentage of floating-point inputs that
+      produce valid, unknown, or invalid outputs. Green outcomes are valid,
+      broken down into finite, infinite, and zero outcomes. Unknown outputs are
+      red. Blue outcomes are invalid (fail precondition or domain errors) and
+      are ignored by Herbie.</figcaption>
```

Comment:
Move the bogosity description above. (Probably call it an outcomes bar or something, not "bogosity" :) )

Reply:
but its such a good word 😉